PhysicsTools/NanoAOD/python/common_cff.py

```diff
@@ -12,9 +12,9 @@ def OVar(valtype, compression=None, doc=None, mcOnly=False,precision=-1):
     elif valtype == bool:  valtype = "bool"
     return cms.PSet( 
                 type = cms.string(valtype),
-                compression = cms.string(compression if compression else "none"),
+                #compression = cms.string(compression if compression else "none"),
```

Comment:
If these two are not used in C++ don't we want to remove them entirely from there?

Reply:
was considering indeed ; done